build-chroot/bump-pinned-bazel.sh

```diff
@@ -6,6 +6,7 @@ set -e -u
 BASEPATH="$(realpath "$(dirname "$0")")"
 
 WORKDIR=$(mktemp -d)
+# shellcheck disable=SC2064
```

Comment:
Can we clarify in the commit message of 8a3bd45 that we're also just silencing some errors, and not just fixing them?

Reply:
Worth noting we only silence false positives - that is, the code is actually written in the optimal way and shellcheck is just overzealous in certain instances.

Reply:
Actually I say that, but it's not entirely true.  However, I'll leave a comment in the source code for that situation.

Reply:
Sorry, yes, I mean -- can we mention in the commit message that some of what the commit does is silencing false positives?

Reply:
```
shell-scripts: Fix existing shellcheck errors 

Also silence some false positives, as well as a valid lint which depends on a separate issue.
```